.github/workflows/build-node.yaml

```diff
@@ -16,7 +16,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [12.x, 14.x, 16.x]
+        node-version: [14.x, 16.x, 18.x]
```

Comment:
```suggestion
        node-version: [12.x, 14.x, 16.x, 18.x]
```

Reply:
`12.x` is already present in `build-node.yml`